### DISCUSSION TITLE
Subqueries using prisma

### DISCUSSION THREAD

**Discussion by afcode123** [prisma/prisma#9407] [Q&A] [answered]
Hi!

Have been using prisma for awhile and liking it a lot. However I can't seem to find any method to do subqueries (where there is no relations) without using raw sql? For example if I want to do a subquery in the same table I am querying with findMany etc. Am I missing something, is there a method to do something like this? If not then any chance doing a feature like this?

**Comment by ryands17** [comment, marked as answer]
@afcode123 👋 

Unfortunately this isn't supported at the moment without using raw queries. There's a similar feature request created so it would be great if you could add a 👍 to this so that we can know the priority 🙂 
https://github.com/prisma/prisma/issues/6506

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
